src/rcc.rs

```diff
@@ -75,20 +109,77 @@ impl CFGR {
         self
     }
 
+    pub fn enable_hsi(mut self, is_enabled: bool) -> Self {
+        self.enable_hsi = Some(is_enabled);
+        self
+    }
+
+    pub fn enable_hsi14(mut self, is_enabled: bool) -> Self {
```

Comment:
We do configuration inside the adc pr (at least in the pr)

Reply:
Ok cool I can remove this.  I guess I just assumed all the RCC stuff would be handled in one place.   Clocks configured before peripherals, etc.